### PR TITLE
fixes #11236 - add root_pass option

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

**Comment by domcleal** [comment]
http://projects.theforeman.org/issues/11236 is the Hammer CLI ticket, this one's for Foreman core only.

Should there be an --ask-* flag too?

**Comment by shlomizadok** [comment]
@domcleal - changed the ticket number (sorry for the confusion) and add --ask-* flag. (thanks for this)

File: lib/hammer_cli_foreman/hostgroup.rb [hunk 1]
```diff
@@ -14,6 +14,14 @@ def self.included(base)
       base.option "--parent", "PARENT_NAME",  _("Name of parent hostgroup")
       base.option "--puppet-classes", "PUPPET_CLASS_NAMES", "",
         :format => HammerCLI::Options::Normalizers::List.new
+      base.option "--root-pass", "ROOT_PASSWORD",  _("Root password")
+      base.option "--ask-root-pass", "ASK_ROOT_PW", "",
+        :format => HammerCLI::Options::Normalizers::Bool.new
+    end
+
+    def self.ask_password
+      prompt = "Enter the root password for the host group: "
```
Comment (domcleal) [inline lib/hammer_cli_foreman/hostgroup.rb]: missing string extraction here

Comment (domcleal) [inline lib/hammer_cli_foreman/hostgroup.rb]: (And avoid extracting the trailing space, please)

**Comment by shlomizadok** [comment]
Extracted

File: lib/hammer_cli_foreman/hostgroup.rb [hunk 2]
```diff
@@ -14,6 +14,14 @@ def self.included(base)
       base.option "--parent", "PARENT_NAME",  _("Name of parent hostgroup")
       base.option "--puppet-classes", "PUPPET_CLASS_NAMES", "",
         :format => HammerCLI::Options::Normalizers::List.new
+      base.option "--root-pass", "ROOT_PASSWORD",  _("Root password")
+      base.option "--ask-root-pass", "ASK_ROOT_PW", "",
+        :format => HammerCLI::Options::Normalizers::Bool.new
+    end
+
+    def self.ask_password
+      prompt = _("Enter the root password for the host group:")
```
Comment (domcleal) [inline lib/hammer_cli_foreman/hostgroup.rb]: Unless `ask` adds it for you, you might need to put the space after the extracted string instead, e.g. `_("Enter:") + " "` so the UI's correctly spaced.

**Comment by shlomizadok** [comment]
Added ```_("Enter:") + " "``` as I don't want to change ```ask``` method

**Comment by tstrachota** [comment]
Works well, merging. Thanks @shlomizadok

**Pull request merged**